tools/backupsecure.py

```diff
@@ -0,0 +1,71 @@
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+import os
+from zipfile import ZipFile
+import mimetypes
+
```

Comment:
Considering that we have this script using the google auth, do we still need to have the backup with selenium?

Reply:
Well, selenium may be able to do the one with authentication, but we haven't figured it out yet